README.md

```diff
@@ -1,4 +1,4 @@
-# RETIRED
+Thgi# RETIRED
```

Comment:
I'm going to reject your pull request because of this typo, but yo successfully submitted! Good job! (will also do a small edit so you can do another PR/Pull Request later if you want)